src/routes/album.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const albumController = require("../controllers/album");
+
+const albumRouter = express.Router();
+
+albumRouter.post("/:artists/:id/albums", albumController.createAlbum);
```

Comment:
This is the tricky route to define as the base route comes off artist router, but the controller is an album controller. There a a couple of options of how to handle this. My preferred way is to change this router to start with `"/albums"`, I would also change `:id` to make it more obvious it refers to the artist id, e.g.
```
albumRouter.post("/albums/:artistId", albumController.createAlbum);
```
note: you would have to change the `id` referenced in your createAlbum function to `artistId` as well for this to work.

You could also move this route into the artist.js file and import the createAlbum controller into there. Let me know if you need help with this.